alfalfa_web/server/api.js

```diff
@@ -38,64 +37,53 @@ class AlfalfaAPI {
     });
   }
```

Comment:
Main thing in this file is making the api more consistent and also having functions operate on the db objects instead of their ids.